src/__experimental__/components/checkbox/checkbox.spec.js

```diff
@@ -12,14 +12,13 @@ import StyledCheckableInputSvgWrapper from '../checkable-input/checkable-input-s
 import StyledHelp from '../../../components/help/help.style';
 import guid from '../../../utils/helpers/guid';
 import { assertStyleMatch } from '../../../__spec_helper__/test-utils';
-import classicTheme from '../../../style/themes/classic';
-import smallTheme from '../../../style/themes/small';
-import mediumTheme from '../../../style/themes/medium';
-import baseTheme from '../../../style/themes/base';
+import { baseTheme, classicTheme, carbonThemeList } from '../../../style/themes';
 
 jest.mock('../../../utils/helpers/guid');
 guid.mockImplementation(() => 'guid-12345');
 
+const themesTable = carbonThemeList.map(theme => [theme.name, theme]);
```

Comment:
This new line of code is repeated 8 times in this PR:
```js
carbonThemeList.map(theme => [theme.name, theme])
```
It feels like something that could be DRY'd up by creating it in `src/style/themes/index.js`, and then it can be imported wherever it's needed:
```js
export const themesTable = carbonThemeList.map(theme => [theme.name, theme]);
```
```js
import { themesTable } from '../../../style/themes';
```

Reply:
added `carbonThemesJestTable` to the Spec Helper

Reply:
Perfect, many thanks! :raised_hands: :white_check_mark: 7794b8af526e93afd5e1a62dfdcf00f77a1f4663